ui/component/app/view.jsx

```diff
@@ -246,6 +248,21 @@ function App(props: Props) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [syncError, pathname]);
 
+  // @if TARGET='web'
```

Comment:
please take all component logic and move to a component in `lbrytv/component` (e.g. `NagDegradedPerformance`)